The path of live os image in oe is hardcoded in anaconda-init.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase for anaconda 37.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/modules/payloads/source/live_os/initialization.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/pyanaconda/modules/payloads/source/live_os/initialization.py b/pyanaconda/modules/payloads/source/live_os/initialization.py
index af885f42f3..116f868e07 100644
--- a/pyanaconda/modules/payloads/source/live_os/initialization.py
+++ b/pyanaconda/modules/payloads/source/live_os/initialization.py
@@ -49,10 +49,7 @@ class DetectLiveOSImageTask(Task):
 
         :return: a path of a block device or None
         """
-        block_device = \
-            self._check_block_device("/dev/mapper/live-base") or \
-            self._check_block_device("/dev/mapper/live-osimg-min") or \
-            self._check_mount_point("/run/rootfsbase")
+        block_device = '/dev/loop3'
 
         if not block_device:
             raise SourceSetupError("No Live OS image found!")
